skimage/transform/_warps.py

```diff
@@ -18,24 +17,38 @@
 )
 
 
+def _multichannel_default(multichannel, ndim):
```

Comment:
Please, add a reminder what should be changed/removed by 0.16 to https://github.com/scikit-image/scikit-image/blob/master/TODO.txt.